snippets/xfs-filesystem-snippet.adoc

```diff
@@ -0,0 +1,21 @@
+// Text snippet included in the following modules:
+//
+// * modules/oadp-1-3-backing-csi-snapshots.adoc
+// * backup_and_restore/application_backup_and_restore/installing/oadp-using-data-mover-for-csi-snapshots-doc.adoc
+
+:_mod-docs-content-type: SNIPPET
+
+[NOTE]
+====
+If you format the volume by using XFS filesystem and the volume is at 100% capacity, the backup fails with a `no space left on device` error. For example:
+
+[source,terminal]
+----
+Error: relabel failed /var/lib/kubelet/pods/3ac..34/volumes/ \
+kubernetes.io~csi/pvc-684..12c/mount: lsetxattr /var/lib/kubelet/ \
+pods/3ac..34/volumes/kubernetes.io~csi/pvc-68..2c/mount/data-xfs-103: \
+no space left on device
+----
+  
+In such a scenario, consider resizing the volume or using a different filesystem type, for example, `ext4`, so that the backup completes successfully.
```

Comment:
```suggestion
In this scenario, consider resizing the volume or using a different filesystem type, for example, `ext4`, so that the backup completes successfully.
```